<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainGraph</class>
 <widget class="QMainWindow" name="mainGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainGraph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1011</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>530</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pathLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="uploadBtn">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listGraphs">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>251</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddGraph">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>530</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRmvGraph">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>530</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove graph</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>567</y>
      <width>300</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Extension (e g .local)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioBtnAll">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>569</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioBtnFile">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>569</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ext</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>560</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXzoom">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>530</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X Zoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnYzoom">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>530</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Zoom</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>graph</zorder>
   <zorder>listGraphs</zorder>
   <zorder>btnAddGraph</zorder>
   <zorder>btnRmvGraph</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>radioBtnAll</zorder>
   <zorder>radioBtnFile</zorder>
   <zorder>btnSave</zorder>
   <zorder>btnXzoom</zorder>
   <zorder>btnYzoom</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1331</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGraphs">
    <property name="title">
     <string>Graphs</string>
    </property>
   </widget>
   <addaction name="menuGraphs"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
